catalog/imaging/catalog-video-transformer/src/main/resources/videoMetcardType.json

```diff
@@ -0,0 +1,320 @@
+      "type":"STRING_TYPE",
+      "stored": true,
+      "indexed": true,
+      "tokenized": false,
+      "multivalued": true
+    },
+    "platform-designation":{
+      "type":"STRING_TYPE",
+      "stored": true,
+      "indexed": true,
+      "tokenized": false,
+      "multivalued": true
+    },
+    "image-source-sensor":{
+      "type":"STRING_TYPE",
+      "stored": true,
+      "indexed": true,
+      "tokenized": false,
+      "multivalued": true
+    },
+    "image-coordinate-system":{
+      "type":"STRING_TYPE",
+      "stored": true,
+      "indexed": true,
+      "tokenized": false,
+      "multivalued": true
+    },
+    "sensor-location":{
+      "type":"GEO_TYPE",
+      "stored": true,
+      "indexed": true,
+      "tokenized": false,
+      "multivalued": true
+    },
+    "sensor-true-altitude":{
+      "type":"DOUBLE_TYPE",
+      "stored": true,
+      "indexed": true,
+      "tokenized": false,
+      "multivalued": true
+    },
+    "slant-range":{
+      "type":"DOUBLE_TYPE",
+      "stored": true,
+      "indexed": true,
+      "tokenized": false,
+      "multivalued": true
+    },
+    "target-width":{
+      "type":"DOUBLE_TYPE",
+      "stored": true,
+      "indexed": true,
+      "tokenized": false,
+      "multivalued": true
+    },
+    "frame-center-location":{
+      "type":"GEO_TYPE",
+      "stored": true,
+      "indexed": true,
+      "tokenized": false,
+      "multivalued": true
+    },
+    "frame-center-elevation":{
+      "type":"DOUBLE_TYPE",
+      "stored": true,
+      "indexed": true,
+      "tokenized": false,
+      "multivalued": true
+    },
+    "offset-corner-location":{
+      "type":"GEO_TYPE",
+      "stored": true,
+      "indexed": true,
+      "tokenized": false,
+      "multivalued": true
+    },
+    "target-location":{
+      "type":"GEO_TYPE",
+      "stored": true,
+      "indexed": true,
+      "tokenized": false,
+      "multivalued": true
+    },
+    "target-location-elevation":{
+      "type":"DOUBLE_TYPE",
+      "stored": true,
+      "indexed": true,
+      "tokenized": false,
+      "multivalued": true
+    },
+    "ground-range":{
+      "type":"DOUBLE_TYPE",
+      "stored": true,
+      "indexed": true,
+      "tokenized": false,
+      "multivalued": true
+    },
+    "platform-call-sign":{
+      "type":"STRING_TYPE",
+      "stored": true,
+      "indexed": true,
+      "tokenized": false,
+      "multivalued": true
+    },
+    "event-start-time":{
+      "type":"DATE_TYPE",
+      "stored": true,
+      "indexed": true,
+      "tokenized": false,
+      "multivalued": true
+    },
+    "operational-mode":{
+      "type":"SHORT_TYPE",
+      "stored": true,
+      "indexed": true,
+      "tokenized": false,
+      "multivalued": true
+    },
+    "corner-location":{
+      "type":"GEO_TYPE",
+      "stored": true,
+      "indexed": true,
+      "tokenized": false,
+      "multivalued": true
+    },
+    "security-classification":{
+      "type":"SHORT_TYPE",
```

Comment:
I think these security attributes should be strings since the integer values aren't meaningful unless you know the spec. Can you update `Stanag4609TransportStreamParser` so it uses `KlvString` for the classifying country and object country codes?